dotCMS/src/main/java/com/dotcms/analytics/metrics/AbstractCondition.java

```diff
@@ -23,8 +23,30 @@ public interface AbstractCondition {
     String value();
 
     enum Operator {
-        EQUALS,
-        CONTAINS
+        EQUALS((value1, value2) -> value1.equals(value2)),
```

Comment:
Maybe this is not a good approach to the EQUALS operator when we are comparing URL, maybe in the URL case we need to not take account the query params or support relative path to not include the domain name or protocol in the comparation.